analyzer/runtime/extract_test.go

```diff
@@ -76,9 +76,14 @@ func TestUnknownError(t *testing.T) {
 }
 
 func TestEmptyError(t *testing.T) {
-	input := &mockError{module: "evm", code: 8, message: "reverted: "}
-	msg := tryParseErrorMessage(input.module, input.code, input.message)
-	require.Nil(t, msg, "tryParseErrorMessage extracted a message when it should not have")
+	inputs := map[string]mockError{
+		DefaultTxRevertErrMsg: {module: "evm", code: 8, message: "reverted: "},
+	}
+	for parsed, input := range inputs {
+		msg := tryParseErrorMessage(input.module, input.code, input.message)
+		require.NotNil(t, msg, "failed to extract default error message")
+		require.Equal(t, parsed, *msg, "tryParseErrorMessage extracted a message that differed from the expected default error message")
+	}
```

Comment:
We don't need a `map` and a `for` loop here, nor do we expect to need it in the future (I think?). So let's keep the simpler, more readable structure without loops.